pkg/model/model.go

```diff
@@ -0,0 +1,10 @@
+package model
+
+// Pokemon entity
+type Pokemon struct {
+	ID       int64  `json:"id"`
+	Name     string `json:"name"`
+	Type     string `json:"type"`
+	Category string `json:"category"`
+	Weakness string `json:"wakness"`
+}
```

Comment:
Let's say you decide to also include Digimons in your project; how you'd handle a new model in your project?

<details> 
  <summary>SPOILER</summary>
Change the name of this file to `pokemon.go`.
</details>